_data/simple-icons.json

```
@@ -15440,6 +15440,11 @@
                 }
             }
         },
+        {
+            "title": "tldraw",
+            "hex": "FFFFFF",
+            "source": "https://tldraw.dev/tldraw-icon.svg"
```

We can use the website as source since the logo can be seen in the footer. We also can add the guidelines mentioned in the original issue.
```suggestion
            "source": "https://tldraw.dev",
            "guidelines": "https://github.com/tldraw/tldraw/blob/main/TRADEMARKS.md"
```